Source/Core/VideoCommon/BPMemory.h

```diff
@@ -777,21 +777,24 @@ struct fmt::formatter<RasColorChan> : EnumFormatter<RasColorChan::Zero>
 
 union TwoTevStageOrders
 {
-  BitField<0, 3, u32> texmap0;  // Indirect tex stage texmap
-  BitField<3, 3, u32> texcoord0;
-  BitField<6, 1, bool, u32> enable0;  // true if should read from texture
-  BitField<7, 3, RasColorChan> colorchan0;
+  BitField<0, 3, u32> texmap_even;
```

Comment:
Just curious, do you find `even` and `odd` more readable or was there a different reason for this?

Reply:
I find it more readable, since 0/1 becomes inaccurate when it's in an array, while even/odd is correct when talking about elements 0/1 (at index 0) and elements 6/7 (at index 3).